src/Routes/Votes.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { voteForArticle } from "../../utils";
+const Votes = (singleArticle) => {
+  const [votes, setVotes] = useState(singleArticle.singleArticle);
+
+  const { article_id } = useParams();
+  console.log(singleArticle);
+  const handleArticleVote = () => {
+    setVotes((curVotes) => {
+      return Number(curVotes + 1);
+    });
+    voteForArticle(article_id).catch(err);
+    if (err) {
```

Comment:
This if statement is not inside the catch block and at the moment is not working as intended. 
We could have a error state that gets set inside the catch block with a boolean, and then render something a bit like you are trying to do here but using a state 🙂  
Feel free to send me a message if something is not clear 🦾